packages/mask/src/plugins/RedPacket/SNSAdaptor/RedPacketERC721Form.tsx

```diff
@@ -273,6 +273,8 @@ export function RedPacketERC721Form(props: RedPacketERC721FormProps) {
 
     const { RED_PACKET_NFT_ADDRESS } = useNftRedPacketConstants(chainId)
 
+    const { value: customCollections = [] } = useNFTRedpacketCustomCollections(account, chainId)
```

Comment:
You are passing it down as a prop, set the fallback to stable EMPTY_LIST.

```suggestion
    const { value: customCollections = EMPTY_LIST } = useNFTRedpacketCustomCollections(account, chainId)
```